src/easter/Makefile

```diff
@@ -0,0 +1,9 @@
+INCLUDE_CJSON=1
+INCLUDE_SHMVAR=1
```

Comment:
Remove line 1-2. You're not using JSON or SHMVARs, so no reason to compile it in.